src/components/PrismicLink.ts

```diff
@@ -204,11 +209,18 @@ export const usePrismicLink = (
 		}
 	});
 
+	const text = computed(() => {
+		const field = unref(props.field);
+
+		return field && "text" in field ? field.text : undefined;
```

Comment:
💡 #idea

@angeloashmore gave me an improvement that is working for me: https://github.com/prismicio/prismic-react/pull/205#discussion_r1735434752

I was able to do something like:
```suggestion
		return isFilled.link(field) ? field.text : undefined;
```

Reply:
@lihbr I could try it out if you want as it's a PR to help us, let me know 🙂

Reply:
This is not possible here because `field` can be a document also which is not compatible with `isFilled.link` (React has two distinct props for that, we only have one in Vue)

Reply:
React 😎